cluster-autoscaler/cloudprovider/kubemark/kubemark_other.go

```diff
@@ -14,8 +16,58 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// +build !linux
+package kubemark
 
-// Dummy implementation. Real one should be build on linux.
+import (
+	apiv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider"
+	"k8s.io/autoscaler/cluster-autoscaler/utils/errors"
+	"k8s.io/kubernetes/plugin/pkg/scheduler/schedulercache"
+)
 
-package kubemark
+type KubemarkCloudProvider struct{}
+
+func BuildKubemarkCloudProvider(kubemarkController *kubemark.KubemarkController, specs []string) (*KubemarkCloudProvider, error) {
+	return nil, cloudprovider.ErrNotImplemented
+}
+
+func (kubemark *KubemarkCloudProvider) Name() string { return "" }
+
+func (kubemark *KubemarkCloudProvider) NodeGroups() []cloudprovider.NodeGroup {
+	return []cloudProvider.NodeGroup{}
+}
+
+func (kubemark *KubemarkCloudProvider) Pricing() (cloudprovider.PricingModel, errors.AutoscalerError) {
+	return nil, cloudprovider.ErrNotImplemented
+}
+
+func (kubemark *KubemarkCloudProvider) NodeGroupForNode(node *apiv1.Node) (cloudprovider.NodeGroup, error) {
+	return nil, cloudprovider.ErrNotImplemented
+}
+
+func (kubemark *KubemarkCloudProvider) GetAvilableMachineTypes() ([]string, error) {
+	return []string{}, cloudprovider.ErrNotImplemented
+}
+
+func (kubemark *KubemarkCloudProvider) NewNodeGroup(name string, machineType string, labels map[string]string, extraResources map[string]resource.Quantity) (cloudprovider.NodeGroup, error) {
+	return nil, cloudprovider.ErrNotImplemented
+}
+
+type FakeNodeGroup struct{}
+
+func (f *FakeNodeGroup) Id() string                            { return "" }
+func (f *FakeNodeGroup) MinSize() int                          { return 1 }
+func (f *FakeNodeGroup) MaxSize() int                          { return 2 }
```

Comment:
Can we make MinSize and MaxSize 0? It's int zero value (like "" for string, false for bool or nil for ptr) and it feels more correct than arbitrary values.